kafka-ui-react-app/src/components/Connect/__tests__/Connect.spec.tsx

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
-import { shallow } from 'enzyme';
+import { render } from 'lib/testHelpers';
 import Connect from 'components/Connect/Connect';
 
 describe('Connect', () => {
   it('matches snapshot', () => {
-    const wrapper = shallow(<Connect />);
-    expect(wrapper).toMatchSnapshot();
+    const wrapper = render(<Connect />);
+    expect(wrapper.baseElement).toMatchSnapshot();
```

Comment:
You can safely remove all snapshot tests (we are getting rid of them, right? @workshur )

Reply:
"Replace" I would say